tests/unit/specs/components/transactions/message-view/__snapshots__/RestakeTxDetails.spec.js.snap

```diff
@@ -33,7 +33,7 @@ exports[`RestakeTxDetails renders a restake transaction message 1`] = `
     </i>
      
     <router-link-stub
-      to="staking/validators/cosmosvaloper18ymm350peujvq2xy9ymyqj4v34ekvnk3tsekuz"
+      to="[object Object]"
```

Comment:
is this expected?

Reply:
seems so. Search for `to=[object Object]`. There are a bunch already like this in snapshots.

But thank you for pointing it out. It's interesting